index.html

```diff
@@ -57,15 +57,9 @@ <h1>Castlevania: Symphony of the Night Randomizer</h1>
                    value="ppf">
             <label for="output-ppf">.ppf</label>
           </div>
-          <div id="target-container" class="pure-u-1">
-            <div>
-              <div id="target" class="hidden">
-                <div>
-                  <span id="status"></span>
-                  <input id="file" type="file" accept=".bin,.iso">
-                </div>
-              </div>
-            </div>
+          <div id="target" class="hidden">
+            <label id="status" for="file"></label>
```

Comment:
Swapped this to `<label>` since it made more sense semantically. It will also trigger the input if clicked.